docs: add v2023.2.1 release notes (#3168)
freifunk-gluon/gluon: docs/releases/v2023.2.1.rst

Gluon v2023.2.1
===============

Added hardware support
----------------------

ath79-generic
~~~~~~~~~~~~~

- Ubiquiti

  - UniFi Swiss Army Knife Ultra

ramips-mt7621
~~~~~~~~~~~~~

- D-Link

  - COVR-X1860 (A1)

Minor changes
-------------

> * Nodes using a fastd VPN connection now report the negotiated method on the Status-Page

Reviewer: ```suggestion
* Nodes using a fastd VPN connection now report the negotiated method on the status page
```

  (`#2465 <https://github.com/freifunk-gluon/gluon/issues/2465>`_)

Bugfixes
--------

* Fixed hostapd being unable to start an AP when selecting channel 116/120 with HT40
  (`#3165 <https://github.com/freifunk-gluon/gluon/issues/3165>`_)

> * Spurious reboot issues on some TP-Link WDR3600 and WDR4300 devices have been fixed

Reviewer: ```suggestion
* Fixed occasional reboot issues on some TP-Link WDR3600 and WDR4300 devices
```
